docs/swagger/spec.yml

```diff
@@ -262,6 +262,83 @@ paths:
           description: 'Unexpected error'
           schema:
             $ref: '#/definitions/Error'
+  /users/{userID}/scopes:
+    x-resource: Users
+    get:
+      summary: 'Get actions on objects a user can operate (a Map(ObjectType, Array[ActionType]))'
+      tags:
+        - Users
+      parameters:
+        - $ref: '#/parameters/userID'
+      responses:
+        200:
+          description: 'User scopes found'
+          schema:
+            $ref: '#/definitions/UserScopes'
+        404:
+          description: 'User scopes not found'
+          schema:
+            $ref: '#/definitions/Error'
+        default:
+          description: 'Unexpected error'
+          schema:
+            $ref: '#/definitions/Error'
+    post:
+      summary: 'Create actions on objects a user can operate (a Map(ObjectType, Array[ActionType]))'
+      tags:
+        - Users
+      parameters:
+        - $ref: '#/parameters/userID'
+        - name: UserScopes
+          in: body
+          required: true
+          schema:
+            $ref: '#/definitions/UserScopes'
+      responses:
+        200:
+          description: 'Actions on objects a user can operate'
+          schema:
+            $ref: '#/definitions/UserScopes'
+        default:
+          description: 'Unexpected error'
+          schema:
+            $ref: '#/definitions/Error'
+    put:
+      summary: 'Update actions on objects a user can operate (a Map(ObjectType, Array[ActionType]))'
```

Comment:
Don't specify the parameters in the description in anything other than plain english unless it's complicated enough to warrant further documentation.